webpack/assets/javascripts/react_app/actions/NotificationActions.js

```diff
@@ -33,7 +33,12 @@ export default {
       expand: group
     });
   },
-  markAsRead(url) {
-    API.markNotificationAsRead(url);
+  markAsRead(id, group) {
```

Comment:
I see your point, although searching for a specific ID inside a specific group is faster then searching for an ID on all the groups...